.gitignore

```diff
@@ -38,3 +38,6 @@ coverage/
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+config/credentials.yml.enc
```

Comment:
This is not supposed to be in the .gitignore file. If it is, I will not have access to the credentials within. This file works in conjunction with the master.key file (which _should_ be in the gitignore and never committed). Rails uses the master.key to decrypt the credentials file so i can access the values stored within, such as api tokens, etc.

Reply:
oooooh, got it. i removed it from the gitignore. thanks!